fritzconnection/tests/test_soaper.py

```diff
@@ -79,9 +79,23 @@ def test_raise_fritzconnection_error(error_code, exception):
     """check for exception raising depending on the error_code"""
     content = content_template.format(error_code=error_code)
     response = Response()
+    response.status_code = 500
```

Comment:
a hard coded status code is not needed here, because the content is `xml`.

Reply:
you're right ... this was a left-over from before